### ISSUE TITLE
Using images in chic-header

### ISSUE THREAD

**Issue by Remillard** [Pablo-Gonzalez-Calderon/chic-header-package#7]
I am rapidly trying to come up to speed on Typst, so I am definitely novice level.  I'm trying to use an image in the header and it's causing some issues.

So for instance, the following:
```
#show: chic.with(
  chic-header(
    left-side: strong([COMPANYNAME]),
    right-side: strong([#title])
  ),
  chic-separator(0.5pt),
  chic-height(1in)
)
```
Works fine and produces:
![image](https://github.com/Pablo-Gonzalez-Calderon/chic-header-package/assets/28012313/c4cbe872-7dab-48c1-be1e-935ba9640646)

However if I replace the left side with:
```
    left-side: image("Header Logo.png"),
```
Then I get some very weird behavior with alignment:
![image](https://github.com/Pablo-Gonzalez-Calderon/chic-header-package/assets/28012313/34a1f6d0-51a9-4635-bdd6-ba5df03c1811)

I figure it has something to do with how an image is anchored, but I couldn't see any options on how to make this happen.  Any ideas?

**Comment by Remillard** [comment]
I have found that adding `align(horizon)` before the content block seems to fix this, at least for the moment.  I don't know if there's a better way to handle it, but this will work for now.

**Comment by Pablo-Gonzalez-Calderon** [comment]
Hi! 😁 That's one way to solve it. Also, there's an option called `v-center` which indicates whether to vertically center all sides of the header or the footer. Here's an example:

```typst
#show: chic.with(
  chic-header(
    v-center: true,
    left-side: strong([COMPANYNAME]),
    right-side: image("logo.png", height: .6in),
  ),
  chic-separator(0.5pt),
  chic-height(1.1in)
)
```

Which will look like this:
![image](https://github.com/Pablo-Gonzalez-Calderon/chic-header-package/assets/30756770/94b4982e-8f3c-4a70-8734-c8243ca9e7da)

Notice that I've also set the logo's height and the header's height to adjust the elements nicely. It will depend on you if you set those properties. In my case, the logo was huge for my `a6` page paper, so I needed to adjust its height.

I hope it helps!

**Comment by Remillard** [comment]
Definitely helps.  I am finding Typst far more accessible than LaTeX, but there's some company formats I'm trying to adhere to.  In some other experiments, I also found that reducing the whitespace margin on the logo itself was quite useful.  I've got some other things to craft before it can be a full solution (revisions table page, etc) but I've been able to get to pretty much everything I need to so far (and by far the most impressive was the friendliness of "tablex".)
